gandalf.js

```diff
@@ -5,72 +5,73 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Consider renaming this variable to something more descriptive for better readability.